docs/multiPhase.md

```diff
@@ -0,0 +1,35 @@
+**Create multi-phase experiment**
```

Comment:
Shall we add more detail about when shall we use multi-phrase?

Reply:
Good idea, added some details.